docs/sequential_rules.rst

```diff
@@ -100,8 +100,10 @@ This rule checks the alignment of the **<=** operators over consecutive sequenti
 
 Following extra configurations are supported:
 
-* :code:`if_control_statements_end_group`,
-* :code:`case_control_statements_end_group`.
+* :code:`if_control_statements_ends_group`,
```

Comment:
The only problem I see with changing `end` with `ends` is the people who already have it configured and using `end`.  VSG should report an error when attempting to configure a rule with an invalid option.  However, is there a better way to communicate this?

We could put it in the release notes, but I doubt many people would read it.

I did add a rule deprication feature so VSG will flag rules that are removed or split up.  Would it be worth it to add option deprication?  If a configuration attempts to change an option that used to exist, `if_control_statements_end_group`, we could report something like:

ERROR:  Rule sequential_005 option if_control_statements_end_group.
   This option has been changed to if_control_statements_ends_group.

Reply:
I believe this is just a documentation issue i.e. the docs have `end`  but the code is correct and uses `ends`

Reply:
I just checked and you are correct.